docs/content/error/$http/baddata.ngdoc

```diff
@@ -0,0 +1,13 @@
+@ngdoc error
+@name $http:baddata
+@fullName Bad JSON Data
+@description
+
+This error occurs when the data parameter passed to the {@link ng.$http `defaultHttpResponseTransform`} service is not a valid JSON object.
```

Comment:
The link should point tot  a specific section (e.g. `ng.$httpProvider.defaults`, `ng.$http.defaults`, or `ng$http#default-transformations`).

BTW, I just noticed that `$httpProvider.defaults.transformRequest` and `$httpProvider.defaults.transformResponse` are not documented. They should be.